content/webapp/views/pages/concepts/concept/concept.helpers.ts

```diff
@@ -75,312 +75,314 @@ type ConceptsConfig = {
+        maxCount: 0,
+      },
+      relatedTopics: {
+        display: true,
+      },
+    },
+    Subject: {
+      displayName: {
+        display: false,
+      },
+      sourcedDescription: {
+        display: false,
+      },
+      partOf: {
+        display: true,
+      },
+      fieldOrArea: {
+        display: false,
+      },
+      imagesBy: {
+        display: false,
+      },
+      imagesIn: {
+        display: false,
+      },
+      imagesAbout: {
+        display: true,
+        label: `Images about ${concept.displayLabel || concept.label}`,
+      },
+      worksBy: {
+        display: false,
+      },
+      worksIn: {
+        display: false,
+      },
+      worksAbout: {
+        display: true,
+        label: 'Works about this subject',
+      },
+      contributors: {
+        maxCount: 4,
+        label: `Top contributors to the collections in ${concept.displayLabel || concept.label}`,
+      },
+      relatedTopics: {
+        display: true,
+      },
+    },
+    Genre: {
+      displayName: {
+        display: true,
+        label: 'Type/technique',
+      },
+      sourcedDescription: {
+        display: true,
+      },
+      partOf: {
+        display: false,
+      },
+      fieldOrArea: {
+        display: false,
+      },
+      imagesBy: {
+        display: false,
+      },
+      imagesAbout: {
+        display: true,
+        label: `Images about ${concept.displayLabel || concept.label}`,
+      },
+      imagesIn: {
+        display: true,
+        label: `Images of ${concept.displayLabel || concept.label}`,
+      },
+      worksBy: {
+        display: false,
+      },
+      worksAbout: {
+        display: true,
+        label: 'Works about this type/technique',
+      },
+      worksIn: {
+        display: true,
+        label: 'Works using this type/technique',
+      },
+      contributors: {
+        maxCount: 4,
+        label: 'Top contributors to the collections using this type/technique',
+      },
+      relatedTopics: {
+        display: true,
+        excludedTopics: ['Person', 'Organisation', 'Agent'],
+      },
+    },
+    Concept: {
+      displayName: {
+        display: true,
+        label: 'Topic',
+      },
+      sourcedDescription: {
+        display: true,
+      },
+      partOf: {
+        display: false,
+      },
+      fieldOrArea: {
+        display: false,
+      },
+      imagesBy: {
+        display: true,
+        label: `Images produced by ${concept.displayLabel || concept.label}`,
+      },
+      imagesAbout: {
+        display: true,
+        label: `Images referencing ${concept.displayLabel || concept.label}`,
+      },
+      imagesIn: {
+        display: false,
+      },
+      worksBy: {
+        display: true,
+        label: 'Works by this topic',
+      },
+      worksAbout: {
+        display: true,
+        label: 'Works referencing this topic',
+      },
+      worksIn: {
+        display: false,
+      },
+      contributors: {
+        maxCount: 0,
```

Comment:
mega np; I still having `display` here might still be nice? Making the check the same everywhere?